### DISCUSSION TITLE
It's possible use SWR with others frameworks?

### DISCUSSION THREAD

**Discussion by FelipeBattistotti** [vercel/swr#1416] [General]
It's possible use SWR with others frameworks? Outside the React ecosystem?

**Comment by shuding** [comment]
Nope, but there're alternatives for other frameworks like https://github.com/Kong/swrv and https://github.com/ConsoleTVs/sswr.

**Reply by souljorje** [reply]
That's pity. Would be nice to have framework-agnostic core!
